### ISSUE TITLE
Keyboard focus jumps to the start of the page after invoking any interactive control in the message area of the bot.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5410]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who rely on keyboard are getting impacted as Keyboard focus is moving to the start of the page after invoking any control in the page. Due to this, the number of keystrokes will be increased to perform the desired action and users are unable to perform their desired action.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Focus management

### How to reproduce the issue?

Step 1: Open the above URL in New Edge/Chrome browsers with valid credentials.
Step 2: Invoke the 'Bot' button which is present under the questions section.
Step 3: Navigate on the page using keyboard tab key.
Step 4: Verify whether Keyboard focus is moving to the start of the page after invoking any control in the page or not.

### What do you expect?

Keyboard focus should not move to the start of the page after invoking any control in the page.
Ex: Keyboard focus should land on the next interactive control which occurs after invoking the first menu item under the questions section (i.e. What is contribution program) Or on the whole section which appears after performing action on control.

### What actually happened?

Keyboard focus jumps to the start of the page after invoking any interactive control in the message area of the bot.
**Observation:** For example, we invoke the first menu item under the questions section (i.e. What is contribution program) then keyboard focus is moving to the starting of the page.

### Do you have any screenshots or recordings to repro the issue?

<img width="937" alt="Image" src="https://github.com/user-attachments/assets/1f6f1e4d-0c52-498b-a895-dc4663509f4e" />

### MAS reference

https://aka.ms/MAS2.4.3

### Adaptive Card JSON

```json

```

**Comment by Kirank915** [comment]
#AIlimited;#A11yMAS;#A11ySev2;#Accessibility;;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#K4W;#MAS2.4.3;#Win11-Edge(Chromium);#WWL:Bug1767693;

**Comment by OEvgeny** [comment]
I see the experience is customized. We don't observe the same happening on vanilla non-customized Web Chat. This suggests that the customizations were the reason this happens.

**Comment by Kirank915** [comment]
We have shared feedback with WWL team and closed the bug.
